README.md

```diff
@@ -26,15 +26,8 @@ $ https://github.com/tamminhdao/tictactoe-java.git
 $ cd tictactoe-java
 ```
 
-### Run the game
+### Build and run
 
-* Open the project in an IDE (e.g. IntelliJ IDEA)
-* Navigate to `src/main/java/com/tictactoe/main.java`
-* Run `main.main()`
-
-
-### Run the tests
-
-* Open the project in an IDE (e.g. IntelliJ IDEA)
-* Navigate to `src/main/java/com/tictactoe/`
-* Run the test suite
\ No newline at end of file
+```
+$ ./build_run.sh
+```
```

Comment:
I really like that you updated the README. A lot of people forget to update the documentation as they write new code 👍